src/weberSsim.ts

```diff
@@ -0,0 +1,201 @@
+/**
+ * Implements Dan Weber's ssim-specific logic.
+ * Copyright (c) 2020 Dan Weber <dan@notatypical.agency>
```

Comment:
I would favor adding something like [all-contributors](https://github.com/all-contributors/all-contributors) and make sure you are listed (and deservedly so!) than adding owners per file since over time it's likely there will be more folks modifying these files.

Reply:
Do you mind if this stays for now?  I'm not opposed to adding AUTHORS in the future.

Reply:
Hm let me think about this one, it's forcing me to think a bit more about the contribution model and I want to make sure I get this right before agreeing to it.

Are your concerns around getting credit for your work or something else?

Reply:
What if we modify the readme credits section instead to include something like:

```md
## 💡 Credits
This project is a direct port of algorithms published by Wang, et al. 2004 on "Image Quality Assessment: From Error Visibility to Structural Similarity". The original Matlab scripts are available here with their datasets. To view the steps taken to validate ssim.js results, check the wiki.

The current (default) implementation uses the Weber algorithm, originally developed by Dan Weber, 2020 at [notatypical.agency](http://notatypical.agency/)
```

I suspect having it in the README will have a larger SEO impact although I'm not too familiar with these things. The documentation would also mention both your algorithm and give you credit for the approach.

Reply:
I love this.  Especially if we make it https :-)

Reply:
Side note -- don't run yarn format-all.  It'll destroy the code base....  [I tested.]

I fixed all of the specific files that needed adjustments.

Reply:
This change has been made in the default algorithm branch.  Since it's a breaking change, I've left it there so we could do two version trees.

Reply:
Got yeah that makes sense, we can run format-all later on, I suspect it's related to `prettier` version changes. So yeah let's remove all copyright mentions here as well (I want to make sure there are no issues, real or perceived, around licensing / ownership) and we can add the README messaging on the follow up PR.

Or feel free to add to the README here as well to mention the algorithm and we can update it later to mention it's the default. 

Whichever way you are more comfortable with I'm fine.

Reply:
I fixed this in the other branch I think, but I'll fix it here too.